venue/views.py

```diff
@@ -547,13 +547,16 @@ def check_profile(request):
         if info['status_code'] == 200 and info['position']:
             resp_status = status.HTTP_200_OK
             response['position'] = info['position']
-            forum_rank = ForumUserRank.objects.get(
-                forum_site=forum,
-                name__iexact=info['position'].strip()
-            )
             allowed = False
-            if forum_rank.allowed or config.TEST_MODE:
-                allowed = True
+            try:
+                forum_rank = ForumUserRank.objects.get(
+                    forum_site=forum,
+                    name__iexact=info['position'].strip()
+                )
+                if forum_rank.allowed or config.TEST_MODE:
+                    allowed = True
+            except ForumUserRank.DoesNotExist:
```

Comment:
This looks to me like we are ignoring the error, and that isn't right. Need to understand why this error occurred and what it means rather than just ignoring it.

Reply:
There is an ORM request made by `name` and `site`. `name` it's not a constant and can't be changed. So, as for me it's a normal situation when the name was changed you can get this line anymore. And update of this line goes in another view.

The same thing I see in the migrations for example in this file `venue/migrations/0006_auto_20180724_1106.py:111`.

Reply:
I don't exactly understand. I do understand that the forum rank names don't change. You need to investigate why in fact this error occurred. I'm guessing that the parsing logic looked for the rank on bitcointalk and then couldn't find it in our list, and that's where the error is coming from.